packages/generator-open-wc/generators/app/index.js

```diff
@@ -1,10 +1,54 @@
+/* eslint-disable no-console */
+
 const Generator = require('yeoman-generator');
+const inquirer = require('inquirer');
+
+const choices = {
+  appStarter: 'Create Open Webcomponents Starter App',
+  appProduction:
+    'Create Open Webcomponents Production Ready App Setup (if you feel lost use the Starter App first)',
```

Comment:
Maybe we should rename it as Open Web Components Enterprise App? I think naming it Production Ready App insinuates the other starter app is not usable to deploy

Reply:
sounds good 👍